libpod/network/create.go

```diff
@@ -80,6 +81,14 @@ func validateBridgeOptions(options entities.NetworkCreateOptions) error {
 
 }
 
+// parseMTU parses the mtu option
+func parseMTU(mtu string) (int, error) {
+	if mtu == "" {
+		return 0, nil // default
+	}
+	return strconv.Atoi(mtu)
```

Comment:
Should we error for negative values? I can do `-o mtu=-1` and it will be added to the config.

Reply:
Good point, that should probably be validated. Supposedly CNI would also do some validation